Lab 1.ipynb

```diff
@@ -0,0 +1,581 @@
+    "ax.set_xlabel(\"Value\", fontsize = 12)\n",
+    "\n",
+    "ax.plot(x, stats.rayleigh.pdf(x, loc = 0, scale = .01), linewidth = 9, alpha = .7)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "40c4ecfe-2184-4517-9d09-da773230c080",
+   "metadata": {},
+   "source": [
+    "Statistical Question: What is the chance that the data at the value \n",
+    "I am looking for a signal at (.028) is due to random variations \n",
+    "as opposed to a real signal?\n",
+    "\n",
+    "Method:<br>\n",
+    "Integrate Rayleigh pdf from hypothetical value to infinity- $ P(x) =  \\int_{0.28}^{\\infty} pdf(x)\\,dx $. Alternatively, could do $ P(x) = 1 - \\int_{-\\infty}^{0.28} pdf(x)\\,dx $ due to normalization.<br>\n",
+    "Then compare to Xσ from normal distribution."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "e33b8186-9647-44de-b6d0-fd0d819f392b",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.019841094744370347"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "hyp_val = .028\n",
+    "\n",
+    "prob = 1 - stats.rayleigh.cdf(hyp_val, loc = 0, scale = 0.01)\n",
+    "prob"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f4495b1a-3fd3-445f-8a68-0cb3d3934617",
+   "metadata": {},
+   "source": [
+    "To get the corresponding Xσ, we use the ppf function from earlier on the *normal* distribution function using the *rayleigh* probability found above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "1ab8b8ad-5241-4afe-8d7d-3200ca8c5c77",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "-2.057041963792887\n"
+     ]
+    }
+   ],
+   "source": [
+    "sigma = stats.norm.ppf(prob, loc = 0, scale = 1)\n",
+    "print(sigma)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "cb81d443-faf4-4c34-84f2-ff88948e4731",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.6065306597126334\n",
+      "0.27028802073873587\n"
+     ]
+    }
+   ],
+   "source": [
+    "hyp_val = .01\n",
+    "\n",
+    "prob = 1 - stats.rayleigh.cdf(hyp_val, loc = 0, scale = 0.01)\n",
+    "print(prob)\n",
+    "\n",
+    "sigma = stats.norm.ppf(prob, loc = 0, scale = 1)\n",
+    "print(sigma)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "28dab91a-8350-42f5-b7cf-ae85813e10bc",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.1175030974154046\n",
+      "-1.1875615473802972\n"
+     ]
+    }
+   ],
+   "source": [
+    "hyp_val = .005\n",
+    "\n",
+    "prob = stats.rayleigh.cdf(hyp_val, loc = 0, scale = 0.01)\n",
```

Comment:
Is there a reason you didn't subtract this probability from 1? Would be nice to either explain that or keep it consistent throughout